core/net/rpl/rpl.c

```diff
@@ -139,7 +139,7 @@ rpl_purge_routes(void)
       PRINTF("No more routes to ");
       PRINT6ADDR(&prefix);
       dag = default_instance->current_dag;
-      /* Propagate this information with a No-Path DAO to preferred parent if we are not a RPL Root */
+      /* Propagate this information with a No-Path DAO to preferred parent if we are not an RPL Root */
```

Comment:
RPL is supposed to be pronounced "ripple", so it should be "a RPL root" and not "an RPL root":